src/components/Categories.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Catagories = () => (
+  <div className="catagories">
+    <button className="status_button" type="button">Check Status</button>
+  </div>
+);
+
+export default Catagories;
```

Comment:
- [ ] Here the "Catagories" is a typo - please, fix it, as the previous reviewer requested to correct one "Categories" (also in Navbar)